workbench/src/renderer/components/SetupTab/SetupButtons/index.jsx

```diff
@@ -1,22 +1,62 @@
 import React from 'react';
 
 import Button from 'react-bootstrap/Button';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
 
 import SaveFileButton from '../../SaveFileButton';
 
+function HoverText(props) {
+  return (
+    <OverlayTrigger
+      placement="right"
+      delay={{ show: 250, hide: 400 }}
+      overlay={(
+        <Tooltip>
+          {props.hoverText}
+        </Tooltip>
+      )}
+    >
+      {/* the first child must be a DOM element, not a custom component,
```

Comment:
Neat that you can comment inside JSX like this!

Reply:
Yes! Sublime knows how to do this just by using `ctrl-/` in a jsx block